### PR TITLE
[+] BO : page -> make searchable

### PR DESCRIPTION
**Pull request by indesign47**

### CODE REVIEW

**Comment by indesign47** [comment]
#47

File: src/Http/Controllers/Backend/PageController.php [hunk 1]
```diff
@@ -29,9 +29,7 @@ class PageController extends Controller
      */
     public function index()
     {
-        $pages = Page::whereType('page')->paginate(15);
-        $type = 'page';
-
+        $pages = Page::search(\Request::get('q'))->paginate(15);
```
Comment (zorx) [inline src/Http/Controllers/Backend/PageController.php:35]: Why did you delete `$type` ?

**Comment by zorx** [comment]
you have override `$pages` in your PR, I'll merge and fix it
